main.js

```diff
@@ -0,0 +1,25 @@
+const tree = new Tree()
+tree.addWord("Artifacts")
+tree.addWord("Aliens")
+tree.addWord("Apple")
+tree.addWord("Apple mac")
+tree.addWord("Apple macbook pro")
+tree.addWord("Apple sucks")
+tree.addWord("Are we in the matrix?")
+tree.addWord("Microsoft")
+tree.addWord("Microsoft windows")
+tree.addWord("Microsoft windows vista sucks")
+tree.addWord("Master Band")
+tree.addWord("Matrix")
```

Comment:
instead, you could make an array of those words and then run a `for` loop (or `forEach`) to add them one by one - cleaner code